es6-classes/square.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+// Square instantiation
+
+class Square extends Shape {
+  constructor(width, area, perimeter) {
+    super(area, perimeter);
```

Comment:
The `constructor` should only have 1 parameter for `width`. Utilizing that `width` parameter, you need to calculate the `area` & `perimeter`